src/main/io/gps.c

```diff
@@ -440,6 +472,25 @@ void gpsInitNmea(void)
 #endif // USE_GPS_NMEA
 
 #ifdef USE_GPS_UBLOX
+const uint8_t ubloxUTCStandardConfig_int[5] = {
+        UBLOX_UTC_STANDARD_AUTO,
+        UBLOX_UTC_STANDARD_USNO,
+        UBLOX_UTC_STANDARD_EU,
+        UBLOX_UTC_STANDARD_SU,
+        UBLOX_UTC_STANDARD_NTSC
+};
+
+char * ubloxVersion_str[] =
+{
+        [UBX_VERSION_UNDEF] = "UNKNOWN",
+        [UBX_VERSION_M5] = "M5",
```

Comment:
```suggestion
struct {
   uint32_t hw;
   const char* str;
} ubloxVersion_map[] = {
        [UBX_VERSION_UNDEF] = {~0, "UNKNOWN"},
        [UBX_VERSION_M5] = {0x00040005, "M5"},
```

Then you can loop over ubloxVersion_map in ubloxDetectVersion. Adding new version would be much easier.